### PR TITLE
Add internal guide on how to run docs site locally

### PR DESCRIPTION
**Pull request by guimachiavelli**
Reviewers: please run the steps on your machines as part of your review.

There might be a few points during the process where you'll see a few warnings and errors, both on terminal and while browsing the local copy of the website. This is expected and shouldn't really influence our testing. Let me know if that's not the case and any errors actually prevent you from working.

### CODE REVIEW

File: running-locally.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   2. Follow instructions to link a Vercel account with access to Meilisearch team
+   3. Choose existing project when requested
+   4. project name: `website`
+6. Fetch config files: `vercel env pull`
+7. Run local server: `yarn dev` and point web browser to `localhost:3000/docs`
+
+### Using a different branch
+
+1. Open `/src/lib/docs/constants.ts`
+2. Search for `BRANCH = main`
```
Comment (maryamsulemani97) [inline running-locally.md:30]:
```suggestion
2. Search for `BRANCH = 'main'`
```

**Review by maryamsulemani97 — COMMENTED** [review]
The steps worked for me for `main`, but I got 3 errors(the docs worked fine):
Screenshot 2023-06-19 at 11 10 26
Screenshot 2023-06-19 at 11 10 54
Screenshot 2023-06-19 at 11 10 50
Screenshot 2023-06-19 at 11 10 45

When I changed the branch to `create-redirects-for-2455`, I got an `Unhandled Runtime Error`  and couldn't access the docs:
Screenshot 2023-06-19 at 11 18 08

File: running-locally.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Requirements
+
+- Node.js v18.x
+- Yarn
+- Vercel CLI: `yarn global add vercel`
+
+## Running a local copy of the docs
+
+By default, this will build use the content at the tip of the `main` branch:
```
[the remark this answers is not in the document]
Comment (guimachiavelli) [reply running-locally.md:13]:
```suggestion
By default, this will use the content at the tip of the `main` branch:
```

**Comment by maryamsulemani97** [comment]
My bad. I used a branch on `meilisearch.com`. It works for branches on the docs.

**Review by maryamsulemani97 — APPROVED** [review]
👟

**Pull request merged**